db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1733,6 +1733,9 @@ int DBImpl::Level0StopWriteTrigger(ColumnFamilyHandle* column_family) {
 
 Status DBImpl::Flush(const FlushOptions& flush_options,
                      ColumnFamilyHandle* column_family) {
+  if (manual_compaction_paused_.load(std::memory_order_acquire) > 0) {
+    return Status::Incomplete(Status::SubCode::kManualCompactionPaused);
```

Comment:
Should we check `flush_option.allow_write_stall` before returning compaction paused?

Reply:
What do you mean? Wait for write stall then check pause?